schemas/2018-05-01/policyDefinition.json

```diff
@@ -0,0 +1,279 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2018-05-01/policyDefinition.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Policy Definition",
+  "description": "This schema defines Azure resource policy definition, please see https://azure.microsoft.com/en-us/documentation/articles/resource-manager-policy/ for more details.",
+  "type": "object",
+  "properties": {
+    "if": {
+      "oneOf": [
+        { "$ref": "#/definitions/condition" },
+        { "$ref": "#/definitions/operatorNot" },
```

Comment:
not a policy expert, but this seems cicular/re-entrant... the oneOfs here are just repeating the definition of the oneOf except for the condition.  If that's indeed valid, not sure why we need 3 of them

Reply:
What each case can contain is a bit different.  It is hard to express it in other ways without looking like a mess.  This is the schema we use in the Ibiza UX JSON editor and I have used it extensively in VS code.  It does parse and validate successfully.

Reply:
Ok - if it's working fine, let's not mess with it ;)